ibis/tests/sql/test_select_sql.py

```diff
@@ -56,7 +56,7 @@ def test_simple_joins(star1, star2):
 
 
 @sqlgolden
-def multiple_joins(con, star1, star2, star3):
```

Comment:
While updating tests, I found a few test functions that didn't start with `test_*` and thus weren't being run. Fixed here.